samza-core/src/main/java/org/apache/samza/runtime/AbstractApplicationRunner.java

```diff
@@ -50,10 +50,12 @@
   private static final Logger log = LoggerFactory.getLogger(AbstractApplicationRunner.class);
 
   private final StreamManager streamManager;
+  private final SystemAdmins systemAdmins;
 
   public AbstractApplicationRunner(Config config) {
     super(config);
-    this.streamManager = new StreamManager(new JavaSystemConfig(config).getSystemAdmins());
+    systemAdmins = new SystemAdmins(config);
```

Comment:
Minor: Use `this.` consistently in the class.

Reply:
Sure. Fixed as suggested.